packages/portalnetwork/package.json

```diff
@@ -9,6 +9,7 @@
     "docs": "typedoc src/index.ts",
     "tape": "tape -r  ts-node/register",
     "test": "tape -r  ts-node/register 'test/!(integration)/**/*.spec.ts'",
+    "test:imtegration": "tape -r ts-node/register 'test/integration/**/*.spec.ts'",
```

Comment:
noticed this typo `test:imtegration` 😄 funny enough the workflow file doesn't use it `- run: npm run tape -- 'test/integration/wire.spec.ts'` so could be updated to `- run: npm run test:integration`

Reply:
also you don't really need a whole new workflow file, an additional job named `test-portalnetwork-integration` in the existing workflow file would have been fine (you could even rename the existing job to `test-portalnetwork-unit), and it would group the checks together as well, sorry just thought of this!

Reply:
Good point. I hadn't thought about that but will definitely simplify the current test setup.  I'll fix my spelling errors too. Planning a cleanup PR for today to take care of several things.